### DISCUSSION TITLE
Library stability

### DISCUSSION THREAD

**Discussion by lucagessi-scmgroup** [rust-embedded/cortex-m#480] [General]
Hi.
I code embedded C for application running on cortex M4, M33 and others.
I would really interested in trying to use rust on embedded applications.
Do you think that this porting is stable enough for industrial applications?
What are the known limitations right now?
Thank you

**Comment by jonathanpallant** [comment]
Framework, the open source laptop people, think so: https://github.com/FrameworkComputer/inputmodule-rs

The Oxide Computer Company also think so: https://github.com/oxidecomputer/hubris

Infineon announced first-party support for some of their microcontrollers too: https://community.infineon.com/t5/Blogs/Infineon-leads-the-way-Enabling-Rust-for-MCUs-in-the-semiconductor-industry/ba-p/410425

Plus I know many more big well-known corporations building Embedded Rust - they just don't talk about it because their work is not open source.
